backend/src/user/user.controller.ts

```
@@ -137,9 +136,7 @@ export class UserController {
             )
             return { message: 'Profile image saved correctly', photoUrl }
         } catch (error) {
-            throw new InternalServerErrorException(
-                'An error occurred when saving the profile image'
-            )
+            console.log('An error occurred when saving the profile image')
```

Should we return a 4xx error to inform to the user a bad use ?